### PR TITLE
pin iconv to 1.15

### PR DESCRIPTION
**Pull request by ocefpaf**

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -22,46 +22,45 @@ requirements:
...
-    - gsl  2.2.*
+    - gsl 2.2.*
     - hdf4
-    # FIXME: We cannot use hdf5 1.10.1 b/c of hdfeos5 is failing with that version.
-    - hdf5 1.8.18|1.8.18.*
+    - hdf5 1.10.1
     - hdfeos2
-    - hdfeos5
+    # FIXME: We cannot use hdfeos5 b/c it won't compiled with hdf5 1.10.1
+    # - hdfeos5
```
Comment (ocefpaf) [inline recipe/meta.yaml]: @khallock is there a way to disable `hdfeos5`? Downgrading to `hdf5 1.8.18` just for that dependency will be quire time consuming and beyond my allocated time as a volunteer for `conda-forge`.

Comment (khallock) [reply recipe/meta.yaml]: @ocefpaf, I just pushed commit f5f8697 to your branch, which should prevent `hdfeos5` support from being built.

Comment (ocefpaf) [reply recipe/meta.yaml]: Thanks! Let's hope that is the only hiccup preventing the build.

**Comment by ocefpaf** [comment]
Closing in favor of #27

**Pull request closed without merging**